src/main/java/org/cascadebot/cascadebot/commandmeta/CommandContext.java

```diff
@@ -128,7 +128,7 @@ public String getMessage(int start, int end) {
         return String.join(" ", ArrayUtils.subarray(args, start, end));
     }
 
-    public boolean isArgInteger(int index) {
+    public boolean isArgNumber(int index) {
```

Comment:
Isn't this a little unnecessarily?

Reply:
Not at all, it's more descriptive.

Reply:
This change is irrelavent because of #178 . @JamesHawkinss could you merge dev into this branch and make sure that isArgNumber is deleted <3